app/profile/components/stats-tabs.tsx

```diff
@@ -155,6 +103,52 @@ export default function StatsTabs({ stats: initialStats }: StatsTabsProps) {
     }
   }
 
+  if (isMobile) {
+    return (
+      <div className="relative">
+        {notification.show && (
+          <CustomNotificationBanner
+            message={notification.message}
+            onClose={() => setNotification({ show: false, message: "" })}
+          />
+        )}
+
+        <div>
+          <div className="h-px bg-gray-200"></div>
+
+          <div
+            onClick={() => router.push("/profile/stats")}
+            className="flex items-center justify-between p-4 cursor-pointer hover:bg-gray-50 transition-colors"
+          >
+            <span className="text-sm font-normal text-gray-900">Stats</span>
+            <Image src="/icons/chevron-right-sm.png" alt="Chevron right" width={20} height={20} />
+          </div>
+
+          <div className="h-px bg-gray-200"></div>
+
+          <div
+            onClick={() => router.push("/profile/payment-methods")}
+            className="flex items-center justify-between p-4 cursor-pointer hover:bg-gray-50 transition-colors"
+          >
+            <span className="text-sm font-normal text-gray-900">Payment methods</span>
+            <Image src="/icons/chevron-right-sm.png" alt="Chevron right" width={20} height={20} />
+          </div>
+
+          <div className="h-px bg-gray-200"></div>
```

Comment:
are these dividers? can we make a separate component for it and just pass it in here and anywhere else we use them, this seems repetitive. We can also pass colour prop to it if we need to change the colour.

```suggestion
          <Divider />
```